tools/packaging/build/agent-enclave-bundle/Dockerfile

```diff
@@ -13,28 +13,31 @@ RUN git clone --depth 1 https://github.com/confidential-containers/enclave-cc.gi
 FROM ubuntu:20.04
 RUN apt-get update && \
     env DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    make \
     wget \
     gnupg
 
 RUN echo "deb [arch=amd64 signed-by=/usr/share/keyrings/intel-sgx.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main" | tee -a /etc/apt/sources.list.d/intel-sgx.list \
  && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | gpg --dearmor --output /usr/share/keyrings/intel-sgx.gpg \
  && apt-get update \
  && env DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    libsgx-enclave-common \
-    libsgx-urts \
-    libsgx-uae-service \
-    libsgx-quote-ex \
     libsgx-dcap-ql \
     libsgx-dcap-quote-verify \
     libsgx-dcap-default-qpl
 
+WORKDIR /opt
+ARG SGX_SDK_INSTALLER=sgx_linux_x64_sdk_2.17.101.1.bin
```

Comment:
@mythi The reason to install sgx sdk here is to support simulatuon mode, right?

Reply:
Correct, I'm moving to use the SDK installer to get the `_sim.so` packages. I'm planning to work on an optimization later that drops unnecessary SDK content.

> Also is it better to use ARG to set the sgx version in the path below as well?

I could maybe add another `ARG` for that. Ideally only one for the version should be enough but annoyingly the SDK filename and the actual release version mismatch (`2.17.101.1` vs `2.17.1`)

Reply:
looking at it again, I'm not sure adding a variable for that URL is useful since `2.17.1` is used in one place only

Reply:
Or just put all the version related URL part as one ARG, such as 
```
ARG SGX_SDK_URL=https://download.01.org/intel-sgx/sgx-linux/2.17.1/distro/ubuntu20.04-server/sgx_linux_x64_sdk_2.17.101.1.bin
```
Then use `basename` to get the installer file name.
```
RUN wget $SGX_SDK_URL \
    && chmod +x $(basename $SGX_SDK_URL)
...
```

Reply:
makes sense, I'll update!

Reply:
updated.